lib/logstash/inputs/beats.rb

```diff
@@ -118,7 +118,7 @@ class LogStash::Inputs::Beats < LogStash::Inputs::Base
 
   # The number of seconds before we raise a timeout. 
   # This option is useful to control how much time to wait if something is blocking the pipeline.
-  config :congestion_threshold, :validate => :number, :default => 5
+  config :congestion_threshold, :validate => :number, :default => 5, :deprecated => "This option is now deprecated, the plugin now use a keep alive mechanism to inform Filebeat"
```

Comment:
I think we should maybe inform that this is now done automatically, like:

`This option is now obsolete since congestion control is done automatically`?